cluster-defns/1.hopsworks.yml

```
@@ -40,6 +40,8 @@ attrs:
     LockPagesInMainMemory: 0
     TotalMemoryConfig: 4G
     NumCPUs: 4
+  flyingduck:
+    memory_gbs: 4
```

cpus variable needs to be set as well now. 

```suggestion
    memory_gbs: 4
    cpus: 4
```